README.md

```diff
@@ -1,2 +1,47 @@
-# Ruby-Group-Capstone
-Ruby Group Capstone
+# Ruby-capstone-project
+
+> Ruby group capstone project : its a project for my catalog
+
+## Built With :hammer_and_wrench:
+
+- **GitHub flow:** Correct use of Branches for deployment and features development.
+- **Linters Check:** Local and Pull Request check for errors, bugs and stylistic errors in code.
+
+## Authors :writing_hand: :busts_in_silhouette:
+
+:man_technologist: **Alzubair Alqaraghuli** :writing_hand:
+
+- GitHub: [@Alzubair98](https://github.com/Alzubair98)
+- Twitter: [@FitZubair](https://twitter.com/FitZubair)
+- LinkedIn: [@Alzubair Alqaraghuli](https://www.linkedin.com/in/alzubair-alqaraghuli-272918233/)
+
+:man_technologist: **Francis Wayungi** :writing_hand:
+
+- GitHub: [@githubhandle](https://github.com/wayungi)
+- Twitter: [@twitterhandle](https://twitter.com/FrancisWayungi)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/francis-wayungi-3aa626231)
+
+:man_technologist: **Shahira Sadat** :writing_hand:
+
+- GitHub: [Shahira Sadat](https://github.com/shahira-sadat)
+- Twitter: [Shahira Sadat](https://twitter.com/SadatShahira)
+- Linkedin: [Shahira Sadat](https://www.linkedin.com/in/shahira-sadat-49b402199)
+- Gmail: [Email](shahira.sadat1@gmail.com)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
+
+## video
+
+[LINK](https://drive.google.com/drive/u/1/folders/1EWHotmbfvoPbZcAKUuqrzTyh_Ylp2fPg)
```

Comment:
- [ ] @team great work on your presentation, however, you forgot the project description part, where you explain the purpose of the project. Kindly add a video describing your project's purpose.

Reply:
fixed